Jenkinsfile

```diff
@@ -69,7 +69,7 @@ pipeline {
                     def data = [
                         "aws_1.7.x": "v1.7.11",
                         "aws_1.8.x": "v1.8.5",
-                        "gce_1.9.x": "v1.9.6",
+                        "gce-1.9.x": "v1.9.6",
```

Comment:
this is subtle. can you explain?

Reply:
You're just too quick. I am testing this for now...
Updating the description ^

Reply:
@bassam - description is updated, PTAL.